// sobi/fesm2015/sobi.js
import { ɵɵdefineInjectable, Injectable, Component, NgModule } from '@angular/core';

class SobiService {
    constructor() {
        console.log("This is a test Library");
    }
}
SobiService.ɵprov = ɵɵdefineInjectable({ factory: function SobiService_Factory() { return new SobiService(); }, token: SobiService, providedIn: "root" });
SobiService.decorators = [
    { type: Injectable, args: [{
                providedIn: 'root'
            },] }
];
SobiService.ctorParameters = () => [];

class SobiComponent {
    constructor() { }
    ngOnInit() {
    }
}
SobiComponent.decorators = [
    { type: Component, args: [{
                selector: 'lib-sobi',
                template: `
    <p>
      sobi works!
    </p>
  `
            },] }
];
SobiComponent.ctorParameters = () => [];

class SobiModule {
}
SobiModule.decorators = [
    { type: NgModule, args: [{
                declarations: [
                    SobiComponent
                ],
                imports: [],
                exports: [
                    SobiComponent
                ]
            },] }
];

/*
 * Public API Surface of sobi
 */

/**
 * Generated bundle index. Do not edit.
 */

export { SobiComponent, SobiModule, SobiService };
