// sobi/bundles/sobi.umd.js
(function (global, factory) {
    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('@angular/core')) :
    typeof define === 'function' && define.amd ? define('sobi', ['exports', '@angular/core'], factory) :
    (global = typeof globalThis !== 'undefined' ? globalThis : global || self, factory(global.sobi = {}, global.ng.core));
}(this, (function (exports, i0) { 'use strict';

    var SobiService = /** @class */ (function () {
        function SobiService() {
            console.log("This is a test Library");
        }
        return SobiService;
    }());
    SobiService.ɵprov = i0.ɵɵdefineInjectable({ factory: function SobiService_Factory() { return new SobiService(); }, token: SobiService, providedIn: "root" });
    SobiService.decorators = [
        { type: i0.Injectable, args: [{
                    providedIn: 'root'
                },] }
    ];
    SobiService.ctorParameters = function () { return []; };

    var SobiComponent = /** @class */ (function () {
        function SobiComponent() {
        }
        SobiComponent.prototype.ngOnInit = function () {
        };
        return SobiComponent;
    }());
    SobiComponent.decorators = [
        { type: i0.Component, args: [{
                    selector: 'lib-sobi',
                    template: "\n    <p>\n      sobi works!\n    </p>\n  "
                },] }
    ];
    SobiComponent.ctorParameters = function () { return []; };

    var SobiModule = /** @class */ (function () {
        function SobiModule() {
        }
        return SobiModule;
    }());
    SobiModule.decorators = [
        { type: i0.NgModule, args: [{
                    declarations: [
                        SobiComponent
                    ],
                    imports: [],
                    exports: [
                        SobiComponent
                    ]
                },] }
    ];

    /*
     * Public API Surface of sobi
     */

    /**
     * Generated bundle index. Do not edit.
     */

    exports.SobiComponent = SobiComponent;
    exports.SobiModule = SobiModule;
    exports.SobiService = SobiService;

    Object.defineProperty(exports, '__esModule', { value: true });

})));
